src/components/TodosFilter/TodosFilter.tsx

```diff
@@ -0,0 +1,76 @@
+import { useContext } from 'react';
+import cn from 'classnames';
+import { FilterOptions } from '../../types/FilterOptions';
+import { TodosContext } from '../../contexts/TodosProvider';
+import { TodoAction } from '../../types/TodoAction';
+
+export const TodosFilter = () => {
+  const { todos, filterOptions, dispatch } = useContext(TodosContext);
+
+  const completedTodosCount
+    = todos.filter(({ completed }) => !completed).length;
+
+  return (
+    <footer className="footer" data-cy="todosFilter">
```

Comment:
Some problems with ui when a lot of todos in list

<img width="1440" alt="Screenshot 2024-01-25 at 19 01 20" src="https://github.com/mate-academy/react_todo-app/assets/26187792/ae747b7f-74a1-4b31-946d-826ece6a8341">